packages/destination-actions/src/destinations/adobe-target/adobeTarget_operations.ts

```diff
@@ -0,0 +1,59 @@
+import { RequestClient } from '@segment/actions-core'
+
+const attributes: { [x: string]: string } = {}
+function getnestedObjects(obj: { [x: string]: any }, objectPath = '') {
+  if (obj.traits) {
+    obj = obj.traits
+  }
+  Object.keys(obj).forEach((key) => {
+    const currObjectPath = objectPath ? `${objectPath}.${key}` : key
+    if (typeof obj[key] !== 'object' && obj[key]) {
+      attributes[currObjectPath] = obj[key].toString()
+    } else {
+      getnestedObjects(obj[key], currObjectPath)
+    }
+  })
+  return attributes
+}
+const objectToQueryString = (object: { [x: string]: { toString: () => string } }) =>
+  Object.keys(object)
+    .map((key) => `profile.${key}=${object[key].toString()}`)
+    .join('&')
+
+export default class adobeTarget {
```

Comment:
```suggestion
export default class AdobeTarget {
```

class names should be capitalized